attest/attest.go

```diff
@@ -360,6 +365,9 @@ var (
 	// need to interact with the TPM1.2 device in ways that have not
 	// yet been implemented.
 	ErrTPM12NotImplemented = errors.New("TPM 1.2 support not yet implemented")
+	// ErrTPM20Required is returned in response to OpenTPM() when the
+	// OpenConfig CommandChannel is set and TPMVersion is TPM1.2.
+	ErrTPM20Required = errors.New("TPMVersion must be TPMVersion20 when using CommandChannel")
```

Comment:
I don't think this needs to be part of the API. Just document that CommandChannel returns an error if TPMVersion isn't TPMVersion20.